tests/onnx_backend_node_test.py

```diff
@@ -702,6 +781,79 @@ def exclude_test_cases(backend_test):
         'test_tfidfvectorizer_tf_onlybigrams_levelempty_cpu',
         'test_tfidfvectorizer_tf_onlybigrams_skip5_cpu',
         'test_tfidfvectorizer_tf_uniandbigrams_skip5_cpu',
+        'test_basic_convinteger_cpu',
+        'test_cast_FLOAT_to_STRING_cpu',
+        'test_cast_STRING_to_FLOAT_cpu',
+        'test_cast_DOUBLE_to_FLOAT16_cpu',
+        'test_cast_FLOAT16_to_DOUBLE_cpu',
+        'test_cast_FLOAT16_to_FLOAT_cpu',
+        'test_cast_FLOAT_to_FLOAT16_cpu',
+        'test_matmulinteger_cpu',
+        'test_prelu_broadcast_cpu',
+        'test_convinteger_with_padding_cpu',
+        'test_convtranspose_3d_cpu',
+        'test_dequantizelinear_cpu',
+        'test_instancenorm_epsilon_cpu',
+        'test_instancenorm_example_cpu',
+        'test_isinf_cpu',
+        'test_isinf_negative_cpu',
+        'test_isinf_positive_cpu',
+        'test_nonmaxsuppression_center_point_box_format_cpu',
```

Comment:
CoreML does have NMS layer, I guess those will be added in a later PR?

Reply:
yup